### DISCUSSION TITLE
Prendere spunto da campagna #iovotopendata

### DISCUSSION THREAD

**Discussion by giuragu** [opendatasicilia/informa-covid19#9] [Idee/Proposte]
Ciao,
durante la call di ieri ho proposto di capire se e in che modo poter prendere spunti dalla campagna del 2015 [http://iovotopendata.opendatasicilia.it/](http://iovotopendata.opendatasicilia.it/).

Ovviamente parliamo di finalità totalmente diverse ma potrebbero esserci dei punti da "riutilizzare" magari.
Ad esempio per stimolare i Comuni a prendere posizione pubblica nella richiesta dei dati, nello schema proposto, nei confronti dei Distretti territoriali ASP di riferimento o delle ASP provinciali di riferimento o direttamente del [DASOE](https://www.qualitasiciliassr.it/) stesso.

Al momento è solo una generica idea da perfezionare. Ci tenevo a condividerla.

**Comment by dennisangemi** [comment]
A proposito di campagne, potremmo prendere spunto da questo [tweet ](https://twitter.com/GiorgiaLodi/status/1381317496716791810?s=20)di Giorgia Lodi: **chiedili anche tu al tuo comune**

**Reply by giuragu** [reply]
Quindi @dennisangemi proponi di spostare la campagna su call to action da proporre al cittadino anziché al Comune.
Quindi le possibili call to action sarebbero:

- Cittadino che chiede al Comune;
- Cittadino che chiede all'ASP;
- Cittadino che chiede al DASOE;
- Comune che chiede all'ASP;
- Comune che chiede al DASOE;

Io ne sceglierei una tra queste dopo aver capito quale realmente potrebbe ottenere un impatto.
